pkg/curatedpackages/packageclient.go

```diff
@@ -46,3 +59,74 @@ func convertBundleVersionToPackageVersion(bundleVersions []packagesv1.SourceVers
 	}
 	return versions
 }
+
+func (p *PackageClient) GeneratePackages() ([]packagesv1.Package, error) {
+	packageNameToPackage := p.getPackageNameToPackage()
+	var packages []packagesv1.Package
+	for _, v := range p.packages {
```

Comment:
Why is this a `v`?

Reply:
No reason. Couldn't think of a better letter. Let me change it.